src/Guide/Views.hs

```diff
@@ -913,3 +948,35 @@ findLinksItem item = findLinksMD item' ++ maybeToList (item^.link)
 -- | Find all Markdown links in /any/ structure, using generics.
 findLinksMD :: Data a => a -> [Url]
 findLinksMD a = [url | MD.LINK url _ <- universeBi a]
+
+data ArchiveOrgResponse =
+  ArchiveOrgResponse { archivedSnapshots :: ArchivedSnapshot
+                     } deriving (Show, Generic)
+
+data ArchivedSnapshot =
```

Comment:
I didn't perform this transformation on data types. I will wait for the JSON parser appear. 
Though I replaced time parser by `parseTimeM` .